scripts/install.sh

```diff
@@ -6,7 +6,7 @@ if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then
 fi
 
 echo "Install Bluemix CLI"
-curl -L public.dhe.ibm.com/cloud/bluemix/cli/bluemix-cli/Bluemix_CLI_0.5.6_amd64.tar.gz > Bluemix_CLI.tar.gz
+curl -L public.dhe.ibm.com/cloud/bluemix/cli/bluemix-cli/Bluemix_CLI_0.6.0_amd64.tar.gz > Bluemix_CLI.tar.gz
```

Comment:
We could use  
`curl -L http://public.dhe.ibm.com/cloud/bluemix/cli/bluemix-cli/latest/Bluemix_CLI_amd64.tar.gz > Bluemix_CLI.tar.gz`?

So that we won't need to change this line every time a new bx version rolls out.